swap/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "swap"
 version = "0.7.0"
-authors = [ "The COMIT guys <hello@comit.network>" ]
+authors = ["The COMIT guys <hello@comit.network>"]
```

Comment:
Please avoid reformatting like this.

Reply:
Done. I reverted the formatting. Rustfmt caused this.

Reply:
Rustfmt is formatting `Cargo.toml` now? That is news to me!

Reply:
Sorry it wasn't rustfmt. It was Even Better TOML extension (https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml#formatting) Interesting that rustfmt doesn't do Cargo.toml files tho. Also found you here :D https://github.com/rust-lang/rustfmt/issues/4091#issuecomment-730801431